joomla-projects/gsoc16_improved-multi-lingual
# Undo associations button

#### Summary of Changes

Undo associations button added, it has the following behaviour:
- Removes target association from reference
- Removes all associations in target
- Saves reference and target
- Resets language switcher
#### Testing Instructions
- Select item from the list-view (with associations)
- Go to language that has association
- Click button
- Either pick the same language to see if it's empty or close view to check on the list view if it's gone

Note: Menu's not tested yet, will test after lunch
